src/test/java/de/retest/recheck/RecheckImplTest.java

```diff
@@ -148,50 +148,22 @@ public boolean canCheck( final Object arg0 ) {
 		}
 	}
 
-	private static class WithinTempDirectoryFileNamerStrategy implements FileNamerStrategy {
-
-		private final Path root;
+	private static class WithinTempDirectoryProjectLayout extends SeparatePathsProjectLayout {
 
-		public WithinTempDirectoryFileNamerStrategy( final Path root ) throws IOException {
-			this.root = root;
-		}
-
-		@Override
-		public FileNamer createFileNamer( final String... baseNames ) {
-			return new FileNamer() {
-
-				@Override
-				public File getFile( final String extension ) {
-					return resolveRoot( baseNames, extension );
-				}
-
-				@Override
-				public File getResultFile( final String extension ) {
-					return resolveRoot( baseNames, extension );
-				}
-			};
+		public WithinTempDirectoryProjectLayout( final Path root ) throws IOException {
```

Comment:
While this name is more descriptive, wouldn't it make more sense to add this constructor to `SeparatePathsProjectLayout ` ?

Reply:
You mean a `SamePaths` ctor in the `SeparatePathsProjectLayout`? :-)